thunder/recipes/base.py

```diff
@@ -1,11 +1,35 @@
+import torch
 from thunder import Recipe, Plugin, DebugOptions, Transform, Executor
 from thunder.core.recipe import Interpreter
+from thunder.executors import nvfuser_available
 from thunder.executors.torch_compile import torch_compile_ex
 from thunder.transforms.prune_prologue_checks import PrunePrologueChecks
-
+from thunder.extend import get_all_executors
 from typing import Any
 
 
+def get_nvfuser_package_for_torch(torch_version) -> str:
+    torch_major_minor = torch_version[:4].replace(".", "")
```

Comment:
Three things:
- I wonder if we should use torch.__version__ and torch.version.cuda here?
- Also I wonder if we should just give typical the package name and "see <link to docs> for details".

The caveats seem to be many for to include in the error message:
- PyTorch has no ABI compatibility between anything, so you do need the latest release pytorch (e.g. 2.7.x) for this to work.
- We don't know which SKUs are supported.
- Let's not error in the error message construction.